src/rpc/blockchain.h

```diff
@@ -41,8 +42,10 @@ void RPCNotifyBlockChange(const CBlockIndex*);
 /** Block description to JSON */
 UniValue blockToJSON(const CBlock& block, const CBlockIndex* tip, const CBlockIndex* blockindex, bool txDetails = false) LOCKS_EXCLUDED(cs_main);
 
+typedef std::vector<CAmount> MempoolHistogramFeeRates;
```

Comment:
It is more complicated because of this line:

```cpp
const CAmount fee_rate{CFeeRate{fee, size}.GetFee(1)}; // Fee rate per byte
```

(https://github.com/bitcoin/bitcoin/pull/21422/files#diff-decae4be02fb8a47ab4557fe74a9cb853bdfa3ec0fa1b515c0a1e5de91f4ad0bR1688)

I think your suggestion seems reasonable but then I don't really know whether there are some edge cases I don't know about.

Reply:
Agree, yes. I've tried changing this and it gave me a bunch of warnings. I'd say ignore this comment unless someone else finds this to be problematic.

Reply:
Ok!